examples/with-markdoc/next.config.js

```diff
@@ -0,0 +1,12 @@
+const withMarkdoc = require('@markdoc/next.js')
+// const path = require('path')
+
+module.exports = withMarkdoc({
+  /**
+   * @see https://markdoc.io/docs/nextjs#options
+   */
+  mode: 'server', // 'static' | 'server',
+  //schemaPath: path.join(__dirname, 'markdoc')
+})({
+  pageExtensions: ['js', 'tsx', 'md'],
```

Comment:
```suggestion
  pageExtensions: ['js', 'ts', 'md', 'mdoc']
```

Reply:
What's `mdoc`?
I can't find any file like `.mdoc`.
If we don't use this file extensions, we should remove it.

And also we must remain `tsx`, because we're using this extension in `pages/_app.tsx`